### PR TITLE
feat: add heco chain

### PR DESCRIPTION
**Pull request by pk-850**
### I have updated the outdated explorer link. Thanks

---

## PR-Codex overview
This PR introduces support for the `Heco` chain in the codebase, including its export and definition, alongside the addition of relevant RPC URLs and block explorer details.

### Detailed summary
- Added `heco` export in `src/chains/index.ts`.
- Created `heco` definition in `src/chains/definitions/heco.ts` with:
  - ID: 128
  - Name: 'Huobi ECO Chain Mainnet'
  - Native currency: HT
  - RPC URLs for HTTP and WebSocket
  - Block explorer details for Heco Scan
  - Marked as not a testnet.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by pk-850** [comment]
I have updated the outdated explorer link. Thanks

**Comment by pk-850** [comment]
I have updated the outdated explorer link. Thanks

File: src/chains/definitions/heco.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  rpcUrls: {
+    default: {
+      http: ['https://http-mainnet.hecochain.com'],
+      webSocket: ['wss://ws-mainnet.hecochain.com'],
+    },
+  },
+  blockExplorers: {
+    default: {
+      name: 'Heco Scan',
+      url: 'https://hecoscan.io',
```
Comment (jxom) [inline src/chains/definitions/heco.ts:16]: Seems to not load for me.

Comment (zzq0826) [reply src/chains/definitions/heco.ts:16]: image

Emmm, it's a bit strange. I can load it normally here. I'll close this PR first and find out where the problem is.

Comment (pk-850) [reply src/chains/definitions/heco.ts:16]: image

Hmm, it's a bit strange. I can load it normally here. I'll close this PR first and find out where the problem is.

**Pull request closed without merging**